apps/api/src/routes/v1_keys_createKey.ts

```diff
@@ -125,19 +125,18 @@ When validating a key, we will return this back to you, so you can clearly ident
                     },
                   }),
                 limit: z.number().int().min(1).openapi({
-                  description: "The total amount of burstable requests.",
+                  description: "The total amount of requests in a given interval.",
```

Comment:
we need to deprecate this field entirely
```suggestion
                  description: "The total amount of requests in a given interval.",
                  deprecated: true
```